src/ResourceManagement/ServerManagement/ServerManagement.Tests/OperationsTest.cs

```diff
@@ -0,0 +1,359 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.ServerManagement;
+using Microsoft.Azure.Management.ServerManagement.Models;
+using Microsoft.Rest;
+using Microsoft.Rest.Azure;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ServerManagement.Tests
+{
+    public class OperationsTest : ServerManagementTestBase
+    {
+        public OperationsTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task MakeSureTestsWork()
+        {
+            // ensure known state before starting.
+            await OneTimeInitialization();
+
+            Assert.True(true, "This should always be true.");
+        }
+
+        [Fact]
+        public async Task GatewayTest()
+        {
+            string GatewayName = "sdktest_gateway_205";
+
+            // ensure known state before starting.
+            await OneTimeInitialization();
```

Comment:
OneTimeInitialization seems poorly named, since it's called in every test.  Consider using a fixture instead